packages/core/src/codewhisperer/indexCommon.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * This module is a subset of `./index.js` except it is for 'common' code only.
+ *
+ * See "Shared vs Common" in our docs/ folder for the meaning of 'common'
```

Comment:
I don't think we should continue this pattern. The fact that it requires explanation is one signal that it's not the right pattern. 

If we need node-specific or web-specific code, create a `fooNode.ts` module or `node/` or `web/` directory. The "common" pattern will be hard to untangle later. We should not continue it.